docs/vulnerability_matching.md

```diff
@@ -1,15 +1,15 @@
 # Vulnerability Matching
 
 This is a high level overview of the interfaces that must be implemented for end-to-end vulnerability matching.  
-A description of the end-to-end process is described as well.  
```

Comment:
My editor automatically strips trailing whitespace so that's why it is changed. When I compare the original file with spaces and my file without spaces I don't see any difference in how it is rendered. Are you sure those spaces need to be there?

https://github.com/quay/claircore/blob/master/docs/vulnerability_matching.md
https://github.com/quay/claircore/blob/f9b2dc2cf9f0650da29d72b2d28124d5951471e4/docs/vulnerability_matching.md

Reply:
For github markdown yes.

Reply:
https://meta.stackexchange.com/questions/40976/what-is-the-reason-for-the-top-secret-two-space-newline-markdown-weirdness

It adds the <br> element in the rendered html.

Reply:
ah ok, I will revert the change